collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -2506,17 +2504,13 @@ public void onAnimationEnd(Animation animation) {
     @Override
     public void onAnimationRepeat(Animation animation) {
         // Added by AnimationListener interface.
-        Timber.i("onAnimationRepeat %s",
-                ((animation == inAnimation) ? "in"
-                        : ((animation == outAnimation) ? "out" : "other")));
+
```

Comment:
While you’re in here, why don’t we remove these no-value comments, too. With a modern IDE, or Javadoc, it’s trivially easy to see which interface a method is associated with.
`// Added by AnimationListener interface.` = noise